README.md

```diff
@@ -778,10 +781,226 @@ main();
+
+
+#### Getting Started from Scratch With Nodemon (for Backend and/or CLI projects)
+
+0. install Node Package Manager(NPM): `curl https://www.npmjs.org/install.sh | sh`
+1. Install Spago and PureScript: `npm i -g spago purescript` 
+2. Create a folder for your project: `mkdir <project folder name>`
+3. Move to the project folder: `cd <project folder name>`
+4. Create your PureScript project with Spago: `spago init`, This also produces a `./src/Main.purs` file which contains some starter code.
+5. Initialize the JavaScript/NPM project `npm init`
+6. Add Nodemon as a dependancy `npm install nodemon` (this is usually a dev dependancy, add the `--save-dev` flag to prevent installation in production or CI environments)
+7. Add a JavaScript file which imports and calls the `main` function from the output of `./src/Main.purs`. This can be placed in the root directory for your project. Traditionally this file is named `index.js`. The `main` function from `Main.purs` can accept arguments, this is useful since the Node runtime will replace environment variables inside of JavaScript. It is recommended to read any environment variables in the JavaScript file and pass them as arguments to `main`. Here is an example JavaScript file:
+
+``` JavaScript
+'use strict'
+
+var Main = require('./output/Main');
+
+function main () {
+    /* 
+     Here we could add variables such as 
+     
+     var baseUrl = process.env.BASE_URL;
+
+     Node will replace `process.env.BASE_URL` 
+     with the string contents of the BASE_URL environment
+     variable at bundle/build time.  
+
+     These variables can be supplied to the Main.main function,
+     however, you will need to change the type to accept variables, by default it is an Effect.
+     You will probably want to make it a function from String -> Effect ()
+  */
+
+  Main.main();
+}
+```
+
+8. At this point we should be able to test our program by running `spago build` followed by `node index.js`, you should see '🍝' as output if this was performed successfully 
+9. Now we want to enable Nodemon, Nodemon will watch for file changes in the dependancy tree and reload our Node program each time there is a change during development. We'll also tell Spago to watch our PureScript source files so that they are compiled, which in turn will trigger Nodemon to reload.
+
+To configure this, add the following script to your `package.json` file:
+
+``` 
+...
+  "scripts": {
+    "dev": "spago build --watch & nodemon \"node index.js\"",
+  },
+...
+```
+
+10. You can now run your development environment by running `npm run dev`
+
+11. For a production build, add the following scripts to your `package.json`: 
+
+``` 
+...
+  "scripts": {
+    "build": "spago build",
+    "start": "node index.js"
+  },
+...
+```
+
+12. To run a production build, you can simply run `npm run build` and to start a production process, call `npm start`
+
+For publishing CLI programs or npm modules, please refer to the npm documentation
```

Comment:
It might be nice to add a link to the relevant sections of the documentation mentioned here so people don't risk getting lost